index.html

```diff
@@ -711,10 +714,9 @@ <h4>createDisclosureData</h4>
                 </li>
               </ol>
             </li>
-
             <li>
-Initialize `bbsMessages` to an array of byte arrays obtained from the
-UTF-8 encoding of the the values in the `nonMandatory` array.
+Initialize `bbsMessages` to an array of byte arrays containing the values in the
+`nonMandatory` array of strings encoded using the UTF-8 [[i18n-glossary]] character encoding.
```

Comment:
Actually, the i18n-glossary links the term, so in respec you'd go:

```suggestion
`nonMandatory` array of strings encoded using the <a>UTF-8</a> <a>character encoding</a>.
```

If Respec complains that the term is undefined, you can add the xref to your document. Instructions are in the glossary [here](https://www.w3.org/TR/i18n-glossary/#respec-how-to-use). Note that you can also use the `[= term =]` markup instead of `<a></a>` if you prefer.